release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -203,9 +203,356 @@ For more information, see xref:../applications/creating_applications/creating-ap
+
+[IMPORTANT]
+====
+Compute nodes must run {op-system-first}
+====
+
+[discrete]
+==== {ibmzProductName} and {linuxoneProductName} notable enhancements
+
+Starting with {product-title} 4.14, Extended Update Support (EUS) is extended to the {ibmzProductName} platform.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
+
+The {ibmzProductName} and {linuxoneProductName} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.
+
+This release introduces support for the following features on {ibmzProductName} and {linuxoneProductName}:
+
+* Assisted Installer with z/VM
+* Installing on a single node
+//* Hosted control planes (Technology Preview)
+* Multi-architecture compute nodes
+* oc-mirror plugin
+
+[discrete]
+==== IBM Secure Execution
+
+{product-title} now supports configuring {op-system-first} nodes for IBM Secure Execution on {ibmzProductName} and {linuxoneProductName} (s390x architecture).
+
+For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_z/installing-ibm-z-kvm.html#installing-rhcos-using-ibm-secure-execution_installing-ibm-z-kvm[Installing {op-system} using IBM Secure Execution]
+
 [id="ocp-4-14-ibm-power"]
 === {ibmpowerProductName}
 
+With this release, {ibmpowerProductName} is now compatible with {product-title} {product-version}. For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on {ibmpowerProductName}]
+* xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on {ibmpowerProductName} in a restricted network]
+
+[IMPORTANT]
+====
+Compute nodes must run {op-system-first}
+====
+
+[discrete]
+==== {ibmpowerProductName} notable enhancements
+
+Starting with {product-title} 4.14, Extended Update Support (EUS) is extended to the {ibmpowerProductName} platform.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
+
+The {ibmpowerProductName} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.
+
+This release introduces support for the following features on {ibmpowerProductName}:
+
+* {ibmpowerProductName} Virtual Server Block CSI Driver Operator (Technology Preview)
+* Installer-provisioned Infrastructure Enablement for {ibmpowerProductName} Virtual Server (Technology Preview)
+* Installing on a single node
+//* Hosted control planes (Technology Preview)
```

Comment:
Hosted Control Planes is in TP (is it being commented out)?

Reply:
Yes it has to be commented out. We will reintroduce it after HCP GA mid November through CM